graph.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+# graph.py
+from collections import defaultdict
+
+
+class Graph():
+    def __init__(self, vertices):
+        self.graph = defaultdict(list)
+        self.verticies = vertices
```

Comment:
Change `self.verticies` to `self.num_vertices`, and `vertices` to `num_vertices`. The `num` is important because this is actually the number of vertices (right?).